### PR TITLE
(Almost) finished problems 49 and 50.

### PR DESCRIPTION
**Pull request by ikhat**
Few questions here. 

- For 49.1, I wasn't sure if you wanted something more detailed. I figure the point you want to get across is they don't have to equal like the row spaces do. Any more detailed explanation is sort of subsumed by the next part. 

- For 50.1, I copy/pasted an explanation for what the range of P is from Problem 44.1. Didn't know if I could just reference that. On the other side of the coin, for 50.3 I just used the result of parts of Problem 43, since writing out the entire derivation of those matrices again would be long. Should I reference the problem by number? 

- For 50.4 I'm not sure what sort of explanation to write because I don't know what they'll know about pivots. Do they know that the number of pivot columns in rref(M) equals the number of linearly independent columns in M?

### CODE REVIEW

File: problemsets-mat223/linearalbegra.tex [hunk 1]
```diff
@@ -3934,15 +3946,54 @@ \section*{Task 2.3: Pat and Jamie}
...
+					\rref(R)=\mat{1&0\\0&1}.
+				\]
+				These matrices have 1 and 2 pivots, respectively. 
+			\end{solution}
+		\item Make a conjecture about how the rank of a transformation and the
+			rank of its corresponding matrix relate. Can you justify your claim?
+			\begin{solution}
+				They are equal. 
+
+
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: You can say that the range and column space are the same (by ...) and so their dimensions are the same. Then you can say the dimension of the column space is the same as the number of pivots in rref. (They know about pivots from the homework).

It should work to add a `\label` and `\ref` for problem numbers/parts. If it doesn't, let me know.

File: problemsets-mat223/linearalbegra.tex [hunk 2]
```diff
@@ -3934,15 +3946,54 @@ \section*{Task 2.3: Pat and Jamie}
...
 	\question
-	Let $\mathcal P$ be projection onto $\Span\{\vec u\}$ where $\vec u=\mat{2\\3}$,
+	Let $\mathcal P$ be projection onto $\Span\Set{\vec u}$ where $\vec u=\mat{2\\3}$,
 	and let $\mathcal R$ be rotation counter-clockwise by $90^\circ$.
 	\begin{parts}
 		\item Describe $\Range(\mathcal P)$ and $\Range(\mathcal R)$.
+			\begin{solution}
+				$\Range(\mathcal P)=\Span\Set{\vec u}$, and $\Range(\mathcal R)=\R^2$.
+
+				For $\mathcal P$, by definition of projections $\mathcal P(\vec x)$
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: -> `by the definition of projection`

File: problemsets-mat223/linearalbegra.tex [hunk 3]
```diff
@@ -3934,15 +3946,54 @@ \section*{Task 2.3: Pat and Jamie}
...
+				is the vector in $\Span\Set{\vec u}$ that is closest to $\vec x$,
+				so in particular $\mathcal P(\vec x) \in \Span\Set{\vec u}$ for
+				all $\vec x\in\R^2$. Therefore $\Range(\mathcal P)\subseteq\Span\Set{\vec u}$.
+
+				On the other hand, $\mathcal P(\alpha \vec u)=\alpha\mathcal P(\vec u)=\alpha\vec u$ 
+				for any scalar $\alpha$, and so $\Range(\mathcal P)=\Span\Set{\vec u}$.
+
+				For $\mathcal Q$, we have that any vector $\vec x\in\R^2$, 
+				$\vec x=\mathcal Q(\vec y)$, where $\vec y$ is the rotation of 
+				$\vec x$ clockwise by $90^\circ$.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Therefore ...

**Comment by ikhat** [comment]
I think something may have gone weird with rebasing the branch there. I hope not. I had to manually fix a merge conflict, but it should be all fine now. I felt a tiny bit out of my depth with the git stuff there. 

Let's finalize this branch (prob49-50), I'll resync everything, and then we can get going on the stuff after that. Apologies if I've screwed something up here.

File: problemsets-mat223/linearalbegra.tex [hunk 4]
```diff
@@ -3954,6 +3954,7 @@ \section*{Task 2.3: Pat and Jamie}
 	\newpage
 \end{lesson}
 	\question
+	\label{problem46}
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: This is not a good name for the label...How about `\label{inducedTransform}`

File: problemsets-mat223/linearalbegra.tex [hunk 5]
```diff
@@ -4127,7 +4128,17 @@ \section*{Task 2.3: Pat and Jamie}
 	\]
 	\begin{parts}
 		\item How does the column space of $P$ relate to the column space of $Q$?
+			\begin{solution}
+				They are not equal.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: + but have the same dimension.

File: problemsets-mat223/linearalbegra.tex [hunk 6]
```diff
@@ -4157,15 +4168,62 @@ \section*{Task 2.3: Pat and Jamie}
...
+				and $\Range(\mathcal Q)$ is 2-dimensional. 
+			\end{solution}
 		\item Let $P$ and $R$ be the matrices corresponding to $\mathcal P$ and
 			$\mathcal R$. What is the rank of $P$ and the rank of $R$?
-		\item Make a conjecture about how the rank of a transformation and
-			the rank of its corresponding matrix relate. Can you justify your claim?
+			\begin{solution}
+				$\Rank(P)=1$ and $\Rank(R)=2$.
+
+				By a previous exercise, $P=\frac{1}{13}\mat{4&6\\6&9}$ and
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Can you do a `ref` to that exercise?

**Comment by siefkenj** [comment]
This is very close! It looks like you handled the merge conflicts just fine. If you view the diff on github, you just have to check that a bunch of stuff didn't get incorrectly deleted.

**Pull request merged**